lambda-views/models/sources.yml

```diff
@@ -0,0 +1,13 @@
+version: 2
```

Comment:
I put this in the `macros` directory accidentally — it should be moved to `models`